cmd/libsnap-confine-private/cgroup-support.h

```diff
@@ -30,4 +31,10 @@
  **/
 void sc_cgroup_create_and_join(const char *parent, const char *name, pid_t pid);
 
+/**
+ * sc_cgroup_is_v2() returns true if running on cgroups v2
+ *
+ **/
+bool sc_cgroup_is_v2(void);
```

Comment:
+1 for defining the prototype correctly :)

Reply:
Haha, I read a while ago about the subtle differences between foo() and foo(void) and it makes me cry a bit